v2/swagger.yaml

```diff
@@ -0,0 +1,629 @@
+      github:
+        $ref: "#/definitions/GitHubDetails"
+      licenses:
+        type: array
+        items:
+          type: string
+      topic_list:
+        type: array
+        items:
+          type: string
+      ecosystem:
+        type: string
+      public_vulnerabilities_count:
+        type: integer
+      private_vulnerabilities_count:
+        type: integer
+      public_vulnerabilities:
+        type: array
+        items:
+          $ref: "#/definitions/Vulnerability"
+        description: Publicly known vulnerability details
+      private_vulnerabilities:
+        type: array
+        items:
+          $ref: "#/definitions/Vulnerability"
+        description: Private vulnerability details, available only to registered users
+      affected_direct_deps:
+        type: array
+        description: Lists all direct dependencies of this package
+        items:
+          type: object
+          properties:
+            package:
+              type: string
+            version:
+              type: string
+      version:
+        type: string
+      osio_user_count:
+        type: integer
+  UserStackInfo:
+    title: Stack statistics
+    description: Stack analyses descriptive statistics
+    properties:
+      unknown_dependencies:
+        type: array
+        items:
+          $ref: "#/definitions/UnknownDeps"
+      ecosystem:
+        type: string
+      unknown_dependencies_count:
+        type: integer
+      analyzed_dependencies:
+        type: array
+        items:
+          $ref: "#/definitions/Dependencies"
+      analyzed_dependencies_count:
+        type: integer
+      license_analysis:
+        $ref: "#/definitions/LicenseAnalysis"
+      distinct_licenses:
+        type: array
+        items:
+          type: string
+      stack_license_conflict:
+        type: boolean
+      dependencies:
+        type: array
+        items:
+          $ref: "#/definitions/KnownDeps"
+      total_licenses:
+        type: integer
+      recommendation_ready:
+        type: boolean
+  RecommendedPackageData:
+    title: Recommended packages data
+    description: Recommended packages data
+    allOf:
+      - $ref: "#/definitions/Dependencies"
+      - properties:
+          confidence_reason:
+            type: number
+            format: float
+          reason:
+            type: string
+          topic_list:
+            type: array
+            items:
+              type: string
+          cooccurrence_count:
+            type: number
+          cooccurrence_probability:
+            type: number
+            format: float
+  StackRecommendation:
+    title: Application recommendation
+    description: Application stack recommendations
+    properties:
+      companion:
+        type: array
+        items:
+          $ref: "#/definitions/RecommendedPackageData"
+      manifest_file_path:
+        type: string
+      usage_outliers:
+        type: array
+        items:
+          type: object
+      missing_packages_pgm:
```

Comment:
where do display this field ? can we remove the fields that not are not required ?

Reply:
Not used, Legacy stuff, carried over from v1. I will fix it.